src/scenario.h

```diff
@@ -302,7 +302,7 @@ typedef struct {
 	char custom_strings[0x8000];
 	uint32 game_ticks_1;
 	rct_ride rides[255];
-	uint16 word_01388698;
+	uint16 word_01388698; // TODO: rename to match RCT2_ADDRESS_SAVED_AGE ???
```

Comment:
There is lots of inconsistency around signage of variables because we don't always know at the time what it should be or just guess each time we use a variable.